pkg/util/winutil/process.go

```diff
@@ -0,0 +1,278 @@
+// +build windows
```

Comment:
nit: I would maybe split this in two or three files: `process.go` (common stuff and utils), `process32.go` (32-bit code) and `process64.go` (64-bit code).